### PR TITLE
Refactor upgrades/test_remoteexecution for pytest

### PR DESCRIPTION
**Pull request by mshriver**
Wasn't able to run the tests locally on master branch, going to ask the component owner for some help.  Pretty straightforward conversion though.

### CODE REVIEW

File: tests/upgrades/test_remoteexecution.py [hunk 1]
```diff
@@ -204,49 +197,44 @@ def test_pre_scenario_remoteexecution_satellite(self):
...
+        add_remote_execution_ssh_key(hostname=client.ip_addr)
+        host = entities.Host().search(query={'search': f'name="{client.hostname}"'})
+        host[0].subnet = sn
+        host[0].update(['subnet'])
+        job = entities.JobInvocation().run(
+            data={
+                'job_template_id': 89,
+                'inputs': {'command': "ls"},
+                'targeting_type': 'static_query',
+                'search_query': f"name = {client.hostname}",
```
Comment (JacobCallahan) [inline tests/upgrades/test_remoteexecution.py:232]: I am now the 'single quotes' guy 🙃

**Review by JacobCallahan — CHANGES_REQUESTED** [review]

**Comment by JacobCallahan** [comment]
@lhellebr can you run these changes locally and add the results in here please?

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: -m pre_upgrade tests/upgrades/test_remoteexecution.py

**Review by JacobCallahan — APPROVED** [review]

**Review by devendra104 — APPROVED** [review]
ACK

**Pull request merged**